src/pages/ProductDetail/index.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import { getProductById } from '../../api/get-productById'
+
+class ProductDetail extends React.Component {
+  state = {
+    isLoading: true, // stop confusing users, what happening until fetch data
+    products: {},
+  }
+
+  async componentDidMount() {
+    const { match } = this.props
+
+    const products = await getProductById(match.params.productId)
+    this.setState({ products, isLoading: false })
```

Comment:
[naming] This is product detail, so it's just single product detail data fetched. Use singular instead of *products*. Or replace with something like productData, productDetailData (even if it is longer) or detailData

Reply:
Ouu, i wanted avoid of this plural in product detail, but unfortunately i did it. My bad. DONE!